src/components/ui/dashboard/add-member-button.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import { useState } from "react";
+import { Plus, Users } from "lucide-react";
+import { cn } from "@/components/lib/utils";
+import { useOrganizationStateStore } from "@/lib/providers/organization-state-store-provider";
+import { useRouter } from "next/navigation";
+
+interface AddMemberButtonProps {
+  onClick?: () => void;
+}
+
+export function AddMemberButton({ onClick }: AddMemberButtonProps) {
+  const router = useRouter();
+  const { currentOrganizationId } = useOrganizationStateStore((state) => state);
+  const [hoveredItem, setHoveredItem] = useState<string | null>(null);
+
+  const handleMouseEnter = (item: string) => {
+    setHoveredItem(item);
+  };
+
+  const handleMouseLeave = () => {
+    setHoveredItem(null);
+  };
+
+  return (
+    <>
+      <button
+        className={cn(
+          "flex items-center rounded-lg border border-border bg-card p-4 text-left transition-all hover:shadow-md",
+          hoveredItem === "member" && "shadow-md"
```

Comment:
Can we use the `hover:` tailwind prefix in these buttons? Shouldn't need to track button hover status in a react state.

Reply:
I think I fixed this. I was trying to push this styling to a .css file in /styles without success. It got quite complicated. I may take another try in the future, but going with this for the time being.